### PR TITLE
Removing gunicorn 19.9.0 version

### PR DESCRIPTION
**Pull request by ranade1**
## What changes were proposed in this pull request?

- Removing gunicorn 19.9.0 version

## How was this patch tested?

- Testing with running Hue on MacBook and RHEL 8.4 unsecure version 

Please review [Hue Contributing Guide](https://github.com/cloudera/hue/blob/master/CONTRIBUTING.md) before opening a pull request.

### CODE REVIEW

**Review by amitsrivastava — APPROVED** [review]

File: desktop/core/ext-py3/gunicorn-20.1.0/docs/source/2019-news.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+- Added pypy3 to list of tested environments
+- Grouped `StopIteration` and `KeyboardInterrupt` exceptions with same body together in Arbiter.run()
+- Added `setproctitle` module to `extras_require` in setup.py
+- Avoid unnecessary chown of temporary files
+- Logging: Handle auth type case insensitively
+- Removed `util.import_module`
+- Removed fallback for `types.SimpleNamespace` in tests utils
+- Use `SourceFileLoader` instead instead of `execfile_`
+- Use `importlib` instead of `__import__` and eval`
+- Fixed eventlet patching
```
Comment (wing2fly) [inline desktop/core/ext-py3/gunicorn-20.1.0/docs/source/2019-news.rst:79]: Might need to upgrade eventlet also

File: desktop/core/ext-py3/gunicorn-20.1.0/gunicorn/workers/geventlet.py [hunk 2]
```diff
@@ -130,7 +130,7 @@ def is_already_handled(self, respiter):
         return super().is_already_handled(respiter)
 
     def init_process(self):
-        self.patch()
+        #self.patch()
```
Comment (wing2fly) [inline desktop/core/ext-py3/gunicorn-20.1.0/gunicorn/workers/geventlet.py:133]: this calls 
--> line 124 above eventlet.monkey_patch(), then calls
--> https://github.com/cloudera/hue/blob/master/desktop/core/ext-py3/eventlet-0.30.2/eventlet/__init__.py#L55, then calls 
--> https://github.com/cloudera/hue/blob/master/desktop/core/ext-py3/eventlet-0.30.2/eventlet/patcher.py#L225-L364

So, I don't feel safe to remove monkey patch in gunicorn.

**Review by wing2fly — CHANGES_REQUESTED** [review]

File: desktop/core/ext-py3/gunicorn-20.1.0/gunicorn/workers/geventlet.py (hunk 2 above)
Comment (wing2fly) [reply desktop/core/ext-py3/gunicorn-20.1.0/gunicorn/workers/geventlet.py:133]: To upgrade eventlet and work with Gunicorn, a fix need to patch gunicorn. Not sure if this is the one you have tried
Issue: https://github.com/benoitc/gunicorn/pull/2581
Fix: https://github.com/benoitc/gunicorn/commit/1299ea9e967a61ae2edebe191082fd169b864c64

Comment (amitsrivastava) [reply desktop/core/ext-py3/gunicorn-20.1.0/gunicorn/workers/geventlet.py:133]: The fix https://github.com/eventlet/eventlet/commit/7aa50ad5 in eventlet was the reason why I pulled in the whole of eventlet library. If the new gunicorn is pulling in eventlet v0.32.0+, then we can do away with our copy of eventlet.

**Pull request closed without merging**